src/distributed_ls/Euclid/_hypre_Euclid.h

```diff
@@ -13,20 +7,17 @@
 #define HYPRE_MODE
 #define OPTIMIZED_DH
 
-#if defined(HYPRE_MODE)
-#include "HYPRE_parcsr_mv.h"
-#include "HYPRE_config.h"
-#include "HYPRE_distributed_matrix_mv.h"
-#include "_hypre_utilities.h"
-
-#elif defined(PETSC_MODE)
-#include "petsc_config.h"
-#endif
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/******************************************************************************
```

Comment:
Don't know why this copyright statement was copied multiple times in this file. But it is fine.

Reply:
This happens because `_hypre_Euclid.h` is generated via the `headers` scripts, which concatenates several header files together. The same happens with `_hypre_parcsr_ls.h` or `_hypre_parcsr_mv.h`, for example.

Reply:
OK. I was wondering if you've used `header` script. It explains. But have you uploaded the script?

Reply:
Yes, I used it. It is already present in master: src/distributed_ls/Euclid/headers